tools/rebuild_provision_request.rb

```diff
@@ -72,6 +72,8 @@
 class Tab
   attr_reader :output, :log
 
+  FIELD_ARRAY = [:security_groups].freeze
```

Comment:
I jokingly want to call this `FIELD_ARRAY_ARRAY` but maybe a more realistic name would be `ARRAY_FIELD_NAMES`?